install_deps.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+echo ""
+echo ""
+echo "=============================================="
+echo "           INSTALLING DEPENDENCIES"
+echo "=============================================="
+echo "Make sure:"
+echo " - Conda env is active"
+echo " - CUDA_HOME is set"
+echo " - HOME_ROBOT_ROOT is set"
+echo "Currently:"
+echo " - CUDA_HOME=$CUDA_HOME"
+echo " - HOME_ROBOT_ROOT=$HOME_ROBOT_ROOT"
+echo " - python=`which python`"
+
```

Comment:
There should be a wait here to confirm that these things look correct to the user?

Reply:
Added a prompt